src/cli/main.go

```diff
@@ -85,10 +91,21 @@ OPTIONS:
 	}
 }
 
-func (as *AutoScaler) Run(cliConnection plugin.CliConnection, args []string) {
-	fmt.Println("Hello AutoScaler")
-}
-
 func main() {
 	plugin.Start(new(AutoScaler))
 }
+
+func (as *AutoScaler) Run(cliConnection plugin.CliConnection, args []string) {
+
+	commands.AutoScaler.CLIConnection = cliConnection
+	parser := flags.NewParser(&commands.AutoScaler, flags.HelpFlag|flags.PassDoubleDash)
+	parser.NamespaceDelimiter = "-"
+
+	_, err := parser.ParseArgs(args)
+	if err != nil {
+		fmt.Printf("Error: %s\n", err.Error())
+		ui.SayFailed()
```

Comment:
say failed first and then print message.

Reply:
but in CF CLI, it states error msg first, i.e.

>> cf app test

App test not found
FAILED

Reply:
cf cli is not consistent... I would prefer to show FAILED first and then the reason.